README.md

```diff
@@ -98,7 +137,7 @@ app.use(bodyParser.urlencoded({extended: true}))
 ```
 
 ```text
-Your answer...
+Telling express to use body-parser and to look at all incoming data as a json object
```

Comment:
👍 It's reading `POST` bodies for data and adding them as key:value pairs to a `.body` property of the request object.